packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -840,15 +856,17 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
                     priceRequest.isGovernance
                 );
 
+            uint256 effectiveStake = voterStake.stake - voterStake.pendingStakes[priceRequest.priceRequestIndex];
```

Comment:
We remove from `voterStake.stake` the pendingStake that is greater than 0 if the voter staked during an active reveal phase where the current price request was active.

Note that this `pendingStake` is not included in the frozen `round.cumulativeActiveStakeAtRound` since we call `_freezeRoundVariables` prior to storing the staked amount. 
In this manner, all slashes are accurately calculated.